karma.py

```diff
@@ -47,11 +47,20 @@ def karma(self, event):
     @cmd
     def top_karma(self, event):
         """Get 5 people with most karma points."""
+        args = event.args
         output = ""
         inverted = [(value, key) for (key, value) in self.karmas.items()]
         karmees = sorted(inverted, reverse=True)
-        # Takes top 5 or less if len(karmees) < 5
-        karmees = karmees[:5]
+
+        n = 5
+        if len(args) == 1:
```

Comment:
@JakubN so the question is what do we do if we have more than one arguments? Should we just check here if we have at least 1 argument?